tests/test_arithmetics.rs

```diff
@@ -256,6 +256,47 @@ fn rhs_arithmetic() {
     py_run!(py, c, "assert 1 ** c == '1 ** RA'");
 }
 
+#[pyclass]
+struct LhsAndRhsArithmetic {}
+
+#[pyproto]
+impl PyNumberProtocol for LhsAndRhsArithmetic {
+    fn __radd__(&self, other: &PyAny) -> PyResult<String> {
+        Ok(format!("{:?} + RA", other))
+    }
+
+    fn __rsub__(&self, other: &PyAny) -> PyResult<String> {
+        Ok(format!("{:?} - RA", other))
+    }
+
+    fn __add__(lhs: &PyAny, rhs: &PyAny) -> PyResult<String> {
+        Ok(format!("{:?} + {:?}", lhs, rhs))
+    }
+
+    fn __sub__(lhs: &PyAny, rhs: &PyAny) -> PyResult<String> {
+        Ok(format!("{:?} - {:?}", lhs, rhs))
+    }
+}
+
+#[pyproto]
+impl PyObjectProtocol for LhsAndRhsArithmetic {
+    fn __repr__(&self) -> PyResult<&'static str> {
+        Ok("BA")
+    }
+}
+
+#[test]
+fn lhs_override_rhs() {
+    let gil = Python::acquire_gil();
+    let py = gil.python();
+
+    let c = PyCell::new(py, LhsAndRhsArithmetic {}).unwrap();
+    py_run!(py, c, "assert c.__radd__(1) == '1 + BA'");
```

Comment:
So if I understand this test, it means that it's never possible to call `fn __radd__` ?

(`fn __radd__` has `RA` in output, this has `BA`.)

Reply:
Oops, this can be a bug.
I'll check it tommorow.

Reply:
Fixed by using [METH_COEXIST](https://docs.python.org/3/c-api/structures.html?highlight=coexist#METH_COEXIST)